views/core/viewAll.hbs

```diff
@@ -0,0 +1,94 @@
+<body class="hold-transition skin-green fixed sidebar-mini">
+<div class="wrapper">
+  {{> template/header}}
+  {{> template/sidebar}}
+
+  <div class="content-wrapper">
+    <section class="content-header">
+      <h1>
+        Voir les membres
+      </h1>
+      <ol class="breadcrumb">
```

Comment:
utiliser le template que j'ai créé

Reply:
Je ne parviens pas à utiliser ton template, il me retourne le message d'erreur : "Cannot read property 'toUpperCase' of undefined"

Reply:
Ca doit être un des paramètres que tu envois que est null ? Bon pas grave, laissons pour l'instant, je fait une tache sur trello. Au fait tu doit résoudre les commentaires au fur et à mesures que tu les lit / fix :à)

Reply:
En effet tu as raison, c'est bon cela fonctionne correctement. J'ai finis de modifier, je réponds aux commentaires et publie tout ca